client/src/redux/actions/twitterActions.js

```diff
@@ -59,3 +61,28 @@ export const connectToTwitter = () => async (dispatch) => {
     console.log('Something didnt go right while connecting..', err);
   }
 };
+
+export const getTwitterHomeTimeline = () => async (dispatch) => {
+  try {
+    const response = await axiosInstance.get(
+      '/api/social/twitter/statuses/home_timeline'
```

Comment:
So your app makes a request to the backend to provide the twitter latests posts which will call the twitter api.
Is there a reason why you wouldn't call Twitter api directly without using your backend?

Reply:
Yes, the app makes a request to backend and the backed will call the twitter API.
For every request to twitter API I also need to send my app specific tokens, otherwise twitter will return an error.
For this specific route i also need user's tokens with the request as i need to get the tweets on that user's home timeline